scripts/bert/index.rst

```diff
@@ -45,4 +45,47 @@ It gets validation accuracy of `88.7% <https://raw.githubusercontent.com/dmlc/we
 It gets RTE validation accuracy of `70.8% <https://raw.githubusercontent.com/dmlc/web-data/master/gluonnlp/logs/bert/finetuned_rte.log>`_
 , whereas the the original Tensorflow implementation give evaluation results 66.4%.
 
-Some other tasks can be modeled with `--task_name` parameter.
\ No newline at end of file
+Some other tasks can be modeled with `--task_name` parameter.
+
+BERT for SQuAD
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+GluonNLP provides the following example script to fine-tune SQuAD with pre-trained
+BERT model.
+
+SQuAD 1.1
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Use the following command to fine-tune the BERT model for SQuAD1.1 dataset.
+
+Note that this will require more than 12G of GPU memory.
+ 
+.. code-block:: console
+
+    $ python finetune_squad.py --optimizer adam --batch_size 12 --lr 3e-5 --epochs 2 --gpu
```

Comment:
Could you also add the instructions on how to download the squad dataset and how to run evaluation?